README.md

```diff
@@ -1,5 +1,100 @@
-# lvm-operator
-Operator that manages Topolvm
+# LVM Operator
 
-# Contents
-- [docs/design](doc/design/)
+LVM Operator is a local storage orchestrator for Kubernetes which manages
```

Comment:
Please reword to The LVM Operator provides the ability to dynamically provision local storage via the topolvm CSI plugin.